### PR TITLE
Sketch weekend HW overview and instructions - version 1

### PR DESCRIPTION
**Pull request by lrbloomfield**

### CODE REVIEW

File: _homework/interaction-design/sketch-symbols.md [hunk 1]
```diff
@@ -4,17 +4,43 @@ assigned: Aug 31, 2018
...
+
+### Nice to Have
+
+- At least 2 different sizes (i.e. large, medium, or small)
+
+
+2nd Deliverable: Design a Bottom Nav Bar
+-------------------------------------------------
+
+For this exercise, you will create a bottom nav bar for a mobile app similar to one of these:
```
Comment (nevanscott) [inline _homework/interaction-design/sketch-symbols.md:33]: I'm guessing there's meant to be a list after this :)

**Review by nevanscott — DISMISSED** [review]
I made a couple of changes, but this is looking great!

Only thing it needs before publishing is the list that's missing. Other than that, approved!

File: _homework/interaction-design/sketch-symbols.md [hunk 2]
```diff
@@ -4,17 +4,43 @@ assigned: Aug 31, 2018
...
 
-Sketch Tutorials
-----------------
 
+1st Deliverable: Design a Button System
+-------------------------------------------------
 
-Exercise
---------
+1. Watch [Creating a Button System with Nested Symbols on Sketch App](https://www.youtube.com/watch?v=_bjqVF7Fvg4&t=794s) from Sketch Together <small>Duration: 30:39</small>
```
Comment (nevanscott) [inline _homework/interaction-design/sketch-symbols.md]: Looks like this link takes you to a particular time in the video. Is that intentional?

If not, you probably want https://www.youtube.com/watch?v=_bjqVF7Fvg4

File: _homework/interaction-design/sketch-symbols.md (hunk 1 above)
Comment (lrbloomfield) [reply _homework/interaction-design/sketch-symbols.md:33]: Yes, I want to add some photos but don't know how to do that. Help, please! :)

File: _homework/interaction-design/sketch-symbols.md (hunk 2 above)
Comment (lrbloomfield) [reply _homework/interaction-design/sketch-symbols.md]: Oh no not intentional. Good catch! Will change it.

**Review by elise2106 — DISMISSED** [review]

**Review by dzc1 — APPROVED** [review]

**Pull request merged**